src/user_defines.txt

```diff
@@ -42,23 +42,13 @@
 ### COMPATIBILITY OPTIONS: ###
 
 # Invert the TX/RX half-duplex pin in the transmitter code. Handset external
-# modules are almost all inverted. ESP32 targets only
+# modules are almost all inverted. ESP32/8286 targets only
 -DUART_INVERTED
 
 # Use a custom baud rate on the receiver for a KISS v1 FC (which runs at 400000) or any other oddball baud
 # If not defined, will default to 420000
 #-DRCVR_UART_BAUD=400000
 
-# Invert the TX pin in the receiver code to allow an inverted RX pin on the
```

Comment:
Are we sure we want to remove these?
Wont that mean that people who are using inverted CRSF on their FC's SBUS pad will no longer be able to retain that setup, and will be forced to change over to actual SBUS?

Reply:
Nope, they chan choose Invertde CRSF instead.

Reply:
I made it so if the define is set then the default protocol in the config is `Inverted CRSF` otherwise it's `CRSF`